lib/rspec/openapi/extractors/hanami.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+require 'dry/inflector'
+require 'hanami'
+
+# Hanami::Router::Inspector original code
```

Comment:
What does this mean?
Did you copy the code from Hanami?

Reply:
I assume it is from https://github.com/hanami/router/blob/97f75b8529574bd4ff23165460e82a6587bc323c/lib/hanami/router/inspector.rb#L13

They are MIT License now https://github.com/hanami/router/blob/97f75b8529574bd4ff23165460e82a6587bc323c/LICENSE.md so the code does not conflict with here.

Reply:
Yes, I took their class for an example, and am referring to the original class, perhaps that's not a very valid comment.

Reply:
Could you replace the comment with a link to `https://github.com/hanami/router/blob/97f75b8529574bd4ff23165460e82a6587bc323c/lib/hanami/router/inspector.rb#L13`?
It might help when Hanami introduces breaking change to `Inspector `